### ISSUE TITLE
Keep the header for `releases.tsv` 

### ISSUE THREAD

**Issue by dan2k3k4** [goba/new_drupal9_projects#1]
In the official [drupal.org releases.tsv](https://drupal.org/files/releases.tsv) file, we have the header:
`created project_machine_name    version project_name`

I'm unsure what effect this would have on this project, but it would be nice to be able to browse the `releases.tsv` with the name header instead of Github automatically styling the first line (a module) as the header?

**Comment by goba** [comment]
Yeah I would not suggest to run the script on the entirety of the drupal.org releases.tsv as it goes back to several years of releases :D That said, we could add a condition in the script to ignore the header, yeah. It would still be needed to manually cut the file to have meaningful results in a reasonable time :D
